yellowbrick/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "yellowbrick" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3edddb4a16ddd0586169fccd0645b2fa78e266c40968f872753909250ac6826c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - cycler >=0.10.0
+    - matplotlib >=2.0.2,!=3.0.0
+    - numpy >=1.13.0
+    - scikit-learn >=0.20
+    - scipy >=1.0.0
+
+test:
+  imports:
+    - yellowbrick
+    - yellowbrick.classifier
+    - yellowbrick.cluster
+    - yellowbrick.contrib
+    - yellowbrick.contrib.classifier
+    - yellowbrick.contrib.missing
+    - yellowbrick.contrib.statsmodels
+    - yellowbrick.datasets
+    - yellowbrick.features
+    - yellowbrick.gridsearch
+    - yellowbrick.model_selection
+    - yellowbrick.regressor
+    - yellowbrick.style
+    - yellowbrick.target
+    - yellowbrick.text
+    - yellowbrick.utils
+  requires:
+    - pytest
```

Comment:
As you are just testing the imports you do not need ``pytest`` here, right?
```suggestion
```

Reply:
Removing this also made the build fail.

Reply:
Can you remove this again, I think the failures were related to the other changes